src/debugAdapter/goDebug.ts

```
@@ -1073,17 +1091,22 @@ class GoDebugSession extends LoggingDebugSession {
 					this.threads.delete(id);
 				});
 
+				if (this.skipStopEventOnce) {
+					this.skipStopEventOnce = false;
+					return;
```

Can you explain why we need to do this?

---

To silently skip the `pause` that we introduce just to set the breakpoint and `continue` right after.